app/views/components/_markdown-toolbar.html.erb

```diff
@@ -0,0 +1,71 @@
+<markdown-toolbar class="app-c-markdown-toolbar" for="<%= html_for %>">
+  <%= render "govuk_publishing_components/components/details", {
+      title: "Headings and sub-headings"
```

Comment:
Which ones are sub-headings?

Reply:
This is the current copy. I will update with the rest of the guidance.